pkg/ngrokapidriver/driver.go

```diff
@@ -34,8 +34,7 @@ type ngrokAPIDriver struct {
 }
 
 func NewNgrokAPIClient(apiKey string, region string) NgrokAPIDriver {
-	// TODO: Add in a unique user agent here
-	config := ngrok.NewClientConfig(apiKey)
+	config := ngrok.NewClientConfig(apiKey, ngrok.WithUserAgent("ngrok-ingress-controller/v1-alpha"))
```

Comment:
In ruby I remember we used to pull the version from a constant that was used throughout the code, any chance there is something similar we could do here? Using our milestones until GA makes sense though in the meantime

Reply:
A clean way that I've seen to do this is to have it be injected at build time as described [here](https://www.digitalocean.com/community/tutorials/using-ldflags-to-set-version-information-for-go-applications) with the use of `ldflags`. I was thinking this would be a good thing to implement once we get some more automation on the release process and using a tag to inject this at build time. If you feel differently though, let me know.